util/logger/logger.go

```diff
@@ -89,14 +93,45 @@ func addFields(event *zerolog.Event, keyvals ...interface{}) *zerolog.Event {
 }
 
 func NewSubLogger(name string, obj fmt.Stringer) *SubLogger {
+	var writers []io.Writer
+	var fl *lumberjack.Logger
+	var logDirectory string
+	var logFilename string
+	maxLogSize := 100
+
+	// console writer
+	writers = append(writers, zerolog.ConsoleWriter{Out: os.Stderr})
+
+	// file writer
+	currentDirectory, err := os.Getwd()
```

Comment:
This will change the current behavior of the project. By starting the node, the working directory changes to the user-defined directory. Look [here](https://github.com/pactus-project/pactus/blob/85f3063fa46ed4c892d1c7b8e7ec318e8cbe108d/cmd/daemon/start.go#L34) for details. Therefore, all logs should be saved inside that directory.

Reply:
@b00f 
I think there's no way to avoid passing `working dir` to `lumberjack` we should store user defined working directory then when the system will start node should read config and lumberjack will use this directory for logging.

Reply:
The system always read from `<working-dir>/pactus.log` .
lumberjack package should not know where to store the log.